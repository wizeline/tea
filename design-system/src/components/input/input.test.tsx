import { newE2EPage } from '@stencil/core/testing';

describe('wz-input', () => {
  it('renders value', async () => {
    const page = await newE2EPage();
    await page.setContent(
      "<wz-input value='Hello, my name is Marty McFly'></wz-input>",
    );
    const screenshot = await page.compareScreenshot();
    expect(screenshot).toMatchScreenshot();
  });

  it('renders disabled', async () => {
    const page = await newE2EPage();
    await page.setContent("<wz-input disabled='false'></wz-input>");
    const screenshot = await page.compareScreenshot();
    expect(screenshot).toMatchScreenshot();
  });
});
