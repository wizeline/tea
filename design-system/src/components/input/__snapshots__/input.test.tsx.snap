// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wz-input renders disabled 1`] = `
<wz-input
  class="hydrated"
  disabled="false"
>
  
  <wz-form-error>
    <div
      class="input-container"
    >
      <input
        type="text"
      />
    </div>
  </wz-form-error>
</wz-input>
`;

exports[`wz-input renders value 1`] = `
<wz-input
  class="hydrated"
  value="Hello, my name is Marty McFly"
>
  
  <wz-form-error>
    <div
      class="input-container"
    >
      <input
        type="text"
      />
    </div>
  </wz-form-error>
</wz-input>
`;
